app.js

```diff
@@ -13,13 +13,32 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
+hbs.registerPartials(__dirname + '/views/partials');
 
-// ...
 
 // Add the route handlers here:
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then(beersFromApi => {
+      res.render('beers', {beersFromApi});
+      //console.log('Beers from the database: ', beersFromApi)
```

Comment:
The route and the API call and its handler look good.
Do you still need this commented log?

Reply:
Yeah, it's not required to use, I can delete commented logs.